ownership-chain/e2e-tests/tests/config.ts

```diff
@@ -32,10 +32,10 @@ export const SELECTOR_LOG_EVOLVED_WITH_EXTERNAL_TOKEN_URI =
 "0xdde18ad2fe10c12a694de65b920c02b851c382cf63115967ea6f7098902fa1c8";
 
 // Asset Metadata Extender Contract
-export const ASSET_METADATA_EXTENDER_ADDRESS = "0x0000000000000000000000000000000000000404";
+export const ASSET_METADATA_EXTENDER_ADDRESS = "0x0000000000000000000000000000000000000405";
 export const ASSET_METADATA_EXTENDER_ABI = AssetMetadataExtender.abi as AbiItem[];
 export const SELECTOR_LOG_EXTENDED_TOKEN_URI_UPDATED =
```

Comment:
I think that the name of this constant should also change according to the updated naming